tests/test_logformatter.py

```diff
@@ -75,26 +85,30 @@ def test_scraped(self):
 
 class LogFormatterSubclass(LogFormatter):
     def crawled(self, request, response, spider):
-        kwargs = super(LogFormatterSubclass, self).crawled(
-        request, response, spider)
+        kwargs = super(LogFormatterSubclass, self).crawled(request, response, spider)
         CRAWLEDMSG = (
-            u"Crawled (%(status)s) %(request)s (referer: "
-            u"%(referer)s)%(flags)s"
+            u"Crawled (%(status)s) %(request)s (referer: %(referer)s) %(flags)s"
         )
+        log_args = kwargs['args']
+        log_args['flags'] = str(request.flags)
         return {
             'level': kwargs['level'],
             'msg': CRAWLEDMSG,
-            'args': kwargs['args']
+            'args': log_args,
         }
 
 
-class LogformatterSubclassTest(LoggingContribTest):
+class LogformatterSubclassTest(unittest.TestCase):
```

Comment:
hey - I think previously this test  was checking that tests pass for LogFormatter subclasses as well; now it only checks test_flags_in_request - is this changed on purpose?

Reply:
Not on purpose, just a mistake :slightly_smiling_face: 
Updated, overriding three tests which expect different values due to the different LogFormatter. Thanks!